packages/inter-protocol/src/psm/psm.js

```diff
@@ -143,17 +146,24 @@ export const start = async (zcf, privateArgs, baggage) => {
   updateMetrics();
 
   /**
-   * @param {Amount<'nat'>} given
+   * @param {Amount<'nat'>} toMint
    */
-  const assertUnderLimit = given => {
-    const anchorAfterTrade = AmountMath.add(
-      anchorPool.getAmountAllocated('Anchor', anchorBrand),
-      given,
-    );
-    AmountMath.isGTE(params.getMintLimit(), anchorAfterTrade) ||
+  const assertUnderLimit = toMint => {
+    const mintedAfter = AmountMath.add(mintedOutstanding, toMint);
+    AmountMath.isGTE(params.getMintLimit(), mintedAfter) ||
       assert.fail(X`Request would exceed mint limit`);
```

Comment:
we don't have a `fail` template tag yet? darn.

Reply:
would that do?

Incidentally, why is this a template at all when it's a simple string? While we're here, how about adding some feedback: (`X``Request would exceed mint limit ${mintLimit.value}`)

Reply:
The fall template tag would abbreviate assert.fail and X:

```js
x > 0 || fail`x too small: ${x}`
```

Reply:
nifty